recipes/spdlog/all/conanfile.py

```diff
@@ -87,6 +88,11 @@ def _disable_werror(self):
         tools.replace_in_file(os.path.join(self._source_subfolder, "cmake", "utils.cmake"), "/WX", "")
 
     def build(self):
+        if Version(self.version) == "1.8.3":
+            # This seems to be a broken https://github.com/gabime/spdlog/pull/1890
+            tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                "find_package(fmt 5.3.0 CONFIG REQUIRED)", "find_package(fmt REQUIRED)")
```

Comment:
I don't understand why it's needed (and to switch to cmake_find_package), it works without patch if config file is generated by `cmake_find_package_multi`, right?

Reply:
I've been doing more digging it might be a conan generator bug but I am not sure yet.

Check the last comment in my PR ... seems like Conan is not modeling the CMake 100% correctly which is why it breaks


I have not finished the review the components yet

Reply:
yes, I was suspecting this kind of issue.

Reply:
No even multi required a patch (it does not work on older builds) theres a fail above when I tried

I'll add the new version that was released with a fix